app/storage/src/test/java/com/fsck/k9/storage/messages/RetrieveFolderOperationsTest.kt

```diff
@@ -45,6 +45,39 @@ class RetrieveFolderOperationsTest : RobolectricTest() {
         assertThat(result).isTrue()
     }
 
+    @Test
+    fun `get folder without sync class`() {
```

Comment:
:+1:

Please make this a test for a typical local folder.

See 
https://github.com/k9mail/k-9/blob/783182a718c5a1d4c94e354f6e73941214addfbc/app/core/src/main/java/com/fsck/k9/mailstore/LocalStore.java#L898-L911

For default values:
https://github.com/k9mail/k-9/blob/783182a718c5a1d4c94e354f6e73941214addfbc/app/storage/src/main/java/com/fsck/k9/storage/StoreSchemaDefinition.java#L84-L102

Reply:
Done